packages/react-core/src/components/Popover/Popover.tsx

```diff
@@ -368,6 +368,24 @@ export const Popover: React.FunctionComponent<PopoverProps> = ({
       }
     }
   };
+  const onTriggerEnter = (event: KeyboardEvent) => {
+    if (event.keyCode === KEY_CODES.ENTER || event.keyCode === KEY_CODES.SPACE) {
+      event.preventDefault();
+      if (triggerManually) {
+        if (visible) {
+          shouldClose(null, hide, event);
+        } else {
+          shouldOpen(show, event);
+        }
+      } else {
+        if (visible) {
+          hide();
+        } else {
+          show(true);
+        }
+      }
+    }
+  };
```

Comment:
I'm wondering if we still need this addition to Popover now that Progress Stepper is using a button. @tlabaj what do you think? I could see it being useful if someone *is* using a Popover on a non-button element at the very least.

Reply:
I've pushed everything except for this suggested change (keep or remove). Let me know what the final verdict is and we should be all good to go on this!